cfme/tests/infrastructure/test_roles_names.py

```diff
@@ -0,0 +1,28 @@
+
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import Quadicon
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = \
+        testgen.infra_providers(metafunc, required_fields=["ssh_credentials"])
+    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist,
+                        scope="module")
+
+ROLES = ['Compute', 'Controller', 'BlockStorage', 'ObjectStorage',
+         'CephStorage']
+
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_roles_name(provider):
+    sel.force_navigate("infrastructure_clusters")
+    my_roles_quads = list(Quadicon.all())
+    result = False
+    for quad in my_roles_quads:
+        role_name = str(quad.name).split('-')[1]
```

Comment:
It is a good question, these names arrives after openstack deployment each
one with its roles

Thanks,

Ronnie Rasouli
Quality Engineer RHOS CloudForms
Red Hat

On Fri, Jul 22, 2016 at 6:06 PM, Pete Savage <notifications@github.com>
wrote:

> In cfme/tests/infrastructure/test_roles_names.py
> <https://github.com/ManageIQ/integration_tests/pull/3204#discussion_r71893974>
> :
>
> > +        testgen.infra_providers(metafunc, required_fields=["ssh_credentials"])
> > +    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist,
> > +                        scope="module")
> > +
> > +ROLES = ['Compute', 'Controller', 'BlockStorage', 'ObjectStorage',
> > +         'CephStorage']
> > +
> > +
> > +
> > +@pytest.mark.usefixtures("setup_provider_modscope")
> > +def test_roles_name(provider):
> > +    sel.force_navigate("infrastructure_clusters")
> > +    my_roles_quads = list(Quadicon.all())
> > +    result = False
> > +    for quad in my_roles_quads:
> > +        role_name = str(quad.name).split('-')[1]
>
> A question if these names are standard for any openstack provider, or
> specific to the environment you have created?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManageIQ/integration_tests/pull/3204/files/bf87338c8bd9f50fcb0ed6d56e5e90b619a9f364#r71893974>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzwYxmqHRFYgRwKhHpyIgU-g8JI5B1Vks5qYNyPgaJpZM4JR8vH>
> .
>